catalogs/catalog-lakehouse-iceberg/src/test/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/integration/test/CatalogIcebergBaseIT.java

```diff
@@ -95,59 +97,36 @@ public class CatalogIcebergIT extends AbstractIT {
   public static String ICEBERG_COL_NAME4 = "iceberg_col_name4";
   private static final String provider = "lakehouse-iceberg";
 
-  private static final ContainerSuite containerSuite = ContainerSuite.getInstance();
-  private static String WAREHOUSE;
-  private static String HIVE_METASTORE_URIS;
-
   private static String SELECT_ALL_TEMPLATE = "SELECT * FROM iceberg.%s";
   private static String INSERT_BATCH_WITHOUT_PARTITION_TEMPLATE =
       "INSERT INTO iceberg.%s VALUES %s";
   private static GravitinoMetalake metalake;
 
   private static Catalog catalog;
 
-  private static HiveCatalog hiveCatalog;
+  private static org.apache.iceberg.catalog.Catalog icebergCatalog;
+  private static org.apache.iceberg.catalog.SupportsNamespaces icebergSupportsNamespaces;
 
   private static SparkSession spark;
 
   @BeforeAll
-  public static void startup() {
+  public void startup() throws Exception {
+    ignoreIcebergRestService = false;
```

Comment:
what's the purpose of `ignoreIcebergRestService` ?

Reply:
when `ignoreIcebergRestService` is true, `IcebergRestService` will not start in `MiniGravitino`.